packages/block-library/src/navigation/edit.js

```diff
@@ -48,17 +50,25 @@ function Navigation( {
 	hasExistingNavItems,
 	hasResolvedPages,
 	isRequestingPages,
+	isRequestingMenuItems,
+	hasResolvedMenus,
+	isRequestingMenus,
 	pages,
+	menus,
+	getMenuItems,
 	setAttributes,
 	setFontSize,
 	updateNavItemBlocks,
 	className,
 } ) {
+	const CREATE_EMPTY_OPTION_VALUE = '__CREATE_EMPTY__';
```

Comment:
I think we should put this outside of the function scope so it isn't recreated every time the function is called.